### DISCUSSION TITLE
Render.bat doesn't point to right directory

### DISCUSSION THREAD

**Discussion by ReinBijlsma** [icosa-gallery/open-brush#181] [Q&A] [answered]
When I use the render.bat to render my scenes in a higher or 360 VR version, it doesn't point to the right directory. Nothing happens. How ( and what ) can I edit in Unity to have the final build point to the right directory, like E: \OpenBrushRein\OpenBrush.exe? I'm not skilled in Unity, so I'd prefer some text file which I can paste in the right script before creating my final build.

**Comment by billyquith** [comment]
Could you [register this as a bug or feature request](https://github.com/icosa-gallery/open-brush/issues) if you think it is one please. This is a discussion forum. Thanks.

**Comment by mikeskydev** [comment, marked as answer]
@ReinBijlsma I believe this is due to the path being hardcoded to the application that initially recorded the video and created `render.bat`, whether that's Tilt Brush, Open Brush, or a specific build of Open Brush. Deleting the files and re-recording from the version you require should re-trigger a new `render.bat` with the correct values. If you need any more support ,we now have a dedicated forum on our Discord: [a support forum is now available on our Discord.](https://discord.gg/W7NCEYnEfy)
